### PR TITLE
Edge release update & GitHub test jobs

### PR DESCRIPTION
**Pull request by JanneKiiskila**
# Mbed Edge Examples pull request
## Description

* Update to latest edge (points to RC1).
* GitHub actions - make & test, Docker build.
* Fix to make Docker build work even from an existing folder used for building.
* Documentation updates.
* Use the maintained misspell in GitHub action.
* Update node simple-pt-example Dockerfile node version.

### Changelog
 
 - [x] `CHANGELOG.md` updated.

### CODE REVIEW

File: .github/workflows/misspell.yml [hunk 1]
```diff
@@ -8,13 +8,26 @@ concurrency:
 
 jobs:
   static-checks:
-    runs-on: ubuntu-latest
+    runs-on: edge-example-ci
```
Comment (petedyerarm) [inline .github/workflows/misspell.yml:11]: What is edge-example-ci ? A selfhosted runner?

Comment (JanneKiiskila) [reply .github/workflows/misspell.yml:11]: Yes, there's two of them running in Mercury now.

File: mqttpt-example/mqttgw_sim/README.md [hunk 2]
```diff
@@ -4,9 +4,12 @@ This is an MQTT gateway simulator to test the `mqttpt-example` protocol translat
 
 ### Requirements
 Requires `mosquitto_pub` client for publishing the MQTT messages to MQTT broker.
+```bash
+sudo apt install mosquitto-clients
```
Comment (petedyerarm) [inline mqttpt-example/mqttgw_sim/README.md:8]:
```suggestion
sudo apt install -y mosquitto-clients
```

**Review by petedyerarm — CHANGES_REQUESTED** [review]

**Review by petedyerarm — APPROVED** [review]
LGTM

**Pull request merged**